source/codegen/metadata_mutation.py

```diff
@@ -14,7 +14,7 @@
     'object', 'operator', 'out', 'override',
     'params', 'private', 'protected', 'public',
     'readonly', 'ref', 'return',
-    'sbyte', 'sealed', 'short', 'sizeof', 'stackalloc', 'static', 'status', 'string', 'struct', 'switch',
```

Comment:
Note: I'm not sure how `status` got on the exclude list. If we did have status as an input or extra output, we would need to special case it, though I'm not sure if how we're doing it is right. For this case, `RegisterDoneEvent` has a status callback param/output that I actually (a) do want to be called `status` and share with the default `status` response param and (b) I need to know it's there.

So I removed it from the list. Nothing else changed. If we hit other status params, they may need other handling. The default behavior is that they'll have a compile error for a local name conflict.

Reply:
Yeah, that sounds fine to me. I'm not sure how `status` got on the list. I'm guessing its from being overly cautious in our initial prototype. 

What you're doing and plans for future `status` params sounds reasonable, though, if they pop up.